student.rb

```diff
@@ -9,4 +11,9 @@ def initialize(age, classroom, name: 'unknown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍